docs/source/templates/layout.html

```diff
@@ -35,6 +35,9 @@
       <a href="https://twitter.com/AlteryxOSS" target="_blank">
         <img class="footer-image-twitter" src="{{ pathto('_static/images/twitter.svg', 1) }}" alt="Twitter">
       </a>
+      <a href="https://join.slack.com/t/alteryx-oss/shared_invite/zt-6inxevps-RSbpr9lsACE1kObXz4rIuA" target="_blank">
```

Comment:
This image formatting needs to be fixed: 
![image](https://user-images.githubusercontent.com/4307001/160920087-8bef043a-6204-4d93-8673-a878a7b72197.png)

Reply:
Removing for now.